src/commands/githubActions/endToEndTest/executeTests.ts

```diff
@@ -174,7 +175,8 @@ async function testPackageInstallFromScratch(
     compose,
     errorLogsTimeout,
     healthCheckUrl,
-    network
```

Comment:
I would instead add the bool var `executeProofOfAttestation: boolean` and set it to true or false where needed. The function `executeTestCheckers` should be agnostic of is update or not. 

Its exactly the same but with a slight difference in syntax definition

Reply:
I agree